### ISSUE TITLE
Inadequate accessibility of the app layouts and improper contrast between various views.

### ISSUE THREAD

**Issue by Hamza417** [TeamNewPipe/NewPipe#11434]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [X] I'm aware that this is a request for NewPipe itself and that requests for adding a new service need to be made at [NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues).
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Feature description

The app should have better contrast between titles and description, and layout should have better spacing around it.

### Why do you want this feature?

To improve accessibility and ease of use, currently it poses various issues over what is emphasized to what degree in the app like titles should be emphasized (bold) and channel names and other info should be de-emphasized (have lower/lighter font tones). It's making it visually difficult to use the app.
